coreppl/coreppl.mc

```diff
@@ -127,7 +127,11 @@ utest expr2str (assume_ (bern_ (float_ 0.7)))
   with "assume (Bern 7.0e-1)" in
 utest expr2str (observe_ (float_ 1.5) (beta_ (float_ 1.0) (float_ 2.0)))
   with "observe 1.50e+0 (Beta 1.0e-0, 2.0e+0)" in
-utest expr2str (weight_ (float_ 1.5)) 
+utest expr2str (weight_ (float_ 1.5))
   with "weight 1.50e+0" in
+utest expr2str (assume_ (categorical_ (seq_ [float_ 0.3, float_ 0.2, float_ 0.5])))
+  with "assume (Categorical [ 3.0e-1,\n  2.0e-1,\n  5.0e-1 ])" in
+utest expr2str (assume_ (multinomial_ (int_ 5) (seq_ [float_ 0.3, float_ 0.2, float_ 0.5])))
+  with "assume (Multinomial 5, [ 3.0e-1,\n  2.0e-1,\n  5.0e-1 ])" in
```

Comment:
These new utests should probably be moved to `dist.mc` instead (with `assume` removed)

Reply:
The problem with that is in `dist.mc`, sequences or const cannot be printed right now (same applies to `DBeta` and `DBernoulli`). Should I keep them in `coreppl.mc` or do you have any suggestion on that?

Reply:
I did the following in my local branch (at the bottom of `dist.mc`):
```
-- Language compositions

lang DistAll = BernDist + BetaDist + ExpDist + EmpiricalDist

lang Test = DistAll + MExprAst + MExprPrettyPrint

mexpr

use Test in

utest expr2str (empirical_ [(1.0, float_ 1.5), (3.0, float_ 1.3)]) with "(Empirical size=2)" in
utest expr2str (exp_ (float_ 1.0)) with "(Exp 1.0e-0)" in

()

```

Maybe I should actually make a PR and merge that one first then?

Reply:
Now added my PR #21.